app/tasks/upload_to_sftp.py

```
@@ -0,0 +1,126 @@
+            "status": "Completed",
+            "file_path": file_path,
+            "sftp_path": remote_path
+        }
+    
+    except Exception as e:
+        # Make sure connections are closed
+        try:
+            if 'sftp' in locals():
+                sftp.close()
+            ssh.close()
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)